doc/install.rst

```diff
@@ -230,6 +230,26 @@ library for Windows, Mac OSX and Linux.
 Anaconda offers scikit-learn as part of its free distribution.
 
 
+Intel conda channel
+-------------------
+
+Intel maintains a dedicated conda channel that ships scikit-learn::
+
+    $ conda install -c intel scikit-learn
+
+This version of scikit-learn comes with alternative solvers for some common
+estimators. Those solvers come from the DAAL C++ library and are optimized for
+multi-core CPUs.
```

Comment:
```suggestion
multi-core intel CPUs.
```
?

Reply:
"Intel CPUs" then. Intel DAAL will take advantage of multiple cores on any architecture, but it is tunes for Intel CPUs specific instruction sets, so using of "Intel CPUs" is justified.

Reply:
Good question. @oleksandr-pavlyk does daal have Intel specific optimizations or should it also work well on compatible hardware (e.g. any amd64 with AVX2 instruction sets and such)?

Reply:
Done.